labels.go

```diff
@@ -37,12 +37,14 @@ type Label struct {
 	ID                     int    `json:"id"`
 	Name                   string `json:"name"`
 	Color                  string `json:"color"`
+	TextColor              string `json:"text_color"` //11.9+
```

Comment:
I appreciate the comments, but I prefer we omit them 🙂

Reply:
Done :-)